### PR TITLE
2.x: Add Completable.andThen(MaybeSource)

### PR DESCRIPTION
**Pull request by vanniktech**

### CODE REVIEW

**Comment by akarnokd** [comment]
There is a compilation error.

**Comment by vanniktech** [comment]
Ah yes import is missing. I still manually stage them and sometimes forget that one is needed since usually every package is already included with `*`.

File: src/main/java/io/reactivex/internal/operators/maybe/MaybeDelayWithCompletable.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        final MaybeSource<T> source;
+
+        OtherObserver(MaybeObserver<? super T> actual, MaybeSource<T> source) {
+            this.actual = actual;
+            this.source = source;
+        }
+
+        @Override
+        public void onSubscribe(Disposable d) {
+            if (DisposableHelper.set(this, d)) {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/internal/operators/maybe/MaybeDelayWithCompletable.java]: setOnce

**Review by akarnokd — CHANGES_REQUESTED** [review]

File: src/main/java/io/reactivex/internal/operators/maybe/MaybeDelayWithCompletable.java (hunk 1 above)
Comment (vanniktech) [reply src/main/java/io/reactivex/internal/operators/maybe/MaybeDelayWithCompletable.java]: Then [this](https://github.com/ReactiveX/RxJava/blob/d85d40ecb1d4c3e02a4bc9589dd7af2900752259/src/main/java/io/reactivex/internal/operators/single/SingleDelayWithCompletable.java#L56) would need to be changed too, right?

Comment (akarnokd) [reply src/main/java/io/reactivex/internal/operators/maybe/MaybeDelayWithCompletable.java]: Yes.

Comment (vanniktech) [reply src/main/java/io/reactivex/internal/operators/maybe/MaybeDelayWithCompletable.java]: Alright done

**Review by akarnokd — APPROVED** [review]

**Pull request merged**